Jenkinsfile

```diff
@@ -623,6 +623,7 @@ otherScalaVersions.each { scalaVersion ->
 mainScalaVersions.each { scalaVersion ->
   otherJavaVersions.each { javaVersion ->
     quickMatrix.add([task: "test-suite-default-esversion", scala: scalaVersion, java: javaVersion, testMinify: "false", testSuite: "testSuite"])
+    quickMatrix.add([task: "test-suite-webassembly", scala: scalaVersion, java: mainJavaVersion, testMinify: "false", testSuite: "testSuite"])
```

Comment:
Is this in the correct commit?

Reply:
Yes this is actually in the correct commit. `Character.toString(int)` is a JDK 11+ only method. We only test it under JDK11+. The non-throwing cases are indirectly tested by other methods of the javalib that delegate to that method, but the throwing cases are really only tested on JDK11+.

Adding an intrinsic for that method without adding JDK11+ for Wasm in the CI would therefore be running blind.